CHANGELOG.md

```diff
@@ -15,6 +15,17 @@ it according to semantic versioning. For example, if your PR adds a breaking cha
 should change the heading of the (upcoming) version to include a major version bump.
 
 -->
+# 5.2.1 (upcoming)
+
+## @rjsf/material-ui
+- Patch `RangeWidget` to support `0` as range slider value
+
+## @rjsf/mui
+- Patch `RangeWidget` to support `0` as range slider value
+
+## Dev / docs / playground
+- Updated the playground's `numbers` example to use a range slider with a valid negative and `0` value
+
```

Comment:
Actually, since I haven't cut the 5.2.0 release please append them onto the associated sections for that release